nock/nock-tests.ts

```diff
@@ -22,6 +22,9 @@ inst = scope.head(str);
 inst = scope.get(str);
 inst = scope.get(str, data);
 
+inst = scope.options(str);
+inst = scope.options(str, data);
```

Comment:
Done. I have also updated tests for other scope methods that uses this third parameter.